bootstrap_salt/config.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pkgutil
+
+import bootstrap_cfn.config
+
+from fabric.api import env
+
+import yaml
+
+
+class MyConfigParser(bootstrap_cfn.config.ConfigParser):
+
+    def base_template(self):
+        ret = super(MyConfigParser, self).base_template()
+        ret.add_mapping('KMS', {'salt': {'key': env.kms_key_id}})
```

Comment:
You mean add an init method and fetch them there. Like:
```self.kms_key_id = get_kms_key_id()```

One thing I wanted to avoid is calling KMS every time we run the ConfigParser. i.e. I only want to get KMS stuff when I run cfn_create.